src/main/java/com/csselect/game/Feature.java

```diff
@@ -104,34 +135,21 @@ public BufferedImage getClassGraph() {
 
     /**
      * Setter for the german name of the feature
+     * @param langCode langCode of the language to set the title for
      * @param germanName the german name
      */
-    public void setGermanName(String germanName) {
-        this.nameMap.put(GERMAN_LANGUAGE_CODE, germanName);
+    public void setName(String langCode, String germanName) {
+        this.nameMap.put(langCode, germanName);
```

Comment:
`142 public void setName(String langCode, String germanName) {`

It still says `germanName`. But should just be `name` if I understand correctly (appears at other sections of the code too).